### PR TITLE
add git autoupdate config in CoolQueue.io

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: ajax/libs/CoolQueue.io/package.json [hunk 1]
```diff
@@ -26,5 +26,13 @@
       "type": "git",
       "url": "https://github.com/peterwilli/CoolQueue.io"
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/peterwilli/CoolQueue.io",
```
Comment (PeterDaveHello) [inline ajax/libs/CoolQueue.io/package.json]: missing trailing `.git`

**Comment by PeterDaveHello** [comment]
There is no tag in its git repo, git autoupdate will not work, maybe you can open an issue in its repo first.

**Comment by LeaYeh** [comment]
this project last commit was two years ago...
hmmmmm.... should i still open an issue?

**Comment by PeterDaveHello** [comment]
Why not?

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by LeaYeh** [comment]
sent an issue and wait for respond.

**Comment by PeterDaveHello** [comment]
You should provide the link to the issue in comment.

**Comment by LeaYeh** [comment]
you mean like this?
 https://github.com/peterwilli/CoolQueue.io/issues/3

**Comment by PeterDaveHello** [comment]
yep

**Pull request closed without merging**